### PR TITLE
HTTP verb constant support for net/http

### PR DESCRIPTION
**Pull request by enzowritescode**
#### Summary
- Update `.gitignore`
- Add instructions to `README.md` for running locally
- Update `openApiSync/openApiSync.go` to support `net/http` HTTP verb constants in
- Update `--skip` flags in `.github/workflows/ci.yml`

#### Ticket Link
N/A

### CODE REVIEW

**Review by lieut-data — CHANGES_REQUESTED** [review]
Thanks, @enzowritescode! This looks good -- would you be willing to update the unit test data at https://github.com/mattermost/mattermost-govet/blob/2703fdd901910a9f90953ef656ad88f0b7c49bff/openApiSync/testdata/src/api/users.go#L12-L13 with one of these new constants so we can keep this working going forward?

File: openApiSync/testdata/src/api/users.go [hunk 1]
```diff
@@ -9,8 +9,8 @@ import (
 )
 
 func (a *API) InitUsers() {
-	a.BaseRoutes.Users.Handle("/ids", a.ApiSessionRequired(getUsersByIds)).Methods("POST")                                                                                     // want `Cannot find /api/v4/userzs/ids method: POST in OpenAPI 3 spec. \(maybe you meant: \[/api/v4/users/ids\]\)`
-	a.BaseRoutes.Groups.Handle("/{group_id:[A-Za-z0-9]+}/{syncable_type:teams|channelz}/{syncable_id:[A-Za-z0-9]+}/link", a.ApiSessionRequired(getUsersByIds)).Methods("POST") // want `Cannot find /api/v4/groups/{group_id}/channelz/{syncable_id}/link method: POST in OpenAPI 3 spec.`
+	a.BaseRoutes.Users.Handle("/ids", a.ApiSessionRequired(getUsersByIds)).Methods(http.MethodPost)                                                                                     // want `Cannot find /api/v4/userzs/ids method: POST in OpenAPI 3 spec. \(maybe you meant: \[/api/v4/users/ids\]\)`
+	a.BaseRoutes.Groups.Handle("/{group_id:[A-Za-z0-9]+}/{syncable_type:teams|channelz}/{syncable_id:[A-Za-z0-9]+}/link", a.ApiSessionRequired(getUsersByIds)).Methods(http.MethodPost) // want `Cannot find /api/v4/groups/{group_id}/channelz/{syncable_id}/link method: POST in OpenAPI 3 spec.`
```
Comment (lieut-data) [inline openApiSync/testdata/src/api/users.go]: Thanks, @enzowritescode -- sorry to bug, but could we have one with "POST" and one with `http.MethodPost`? That way we test both the backwards compatibility and the "new way forward", allowing this tool to continue working with previous release branches.

Comment (enzowritescode) [reply openApiSync/testdata/src/api/users.go]: @lieut-data gotcha! Updated.

**Review by lieut-data — APPROVED** [review]
Sweet, thanks @enzowritescode!

**Comment by lieut-data** [comment]
@phoinixgrr, would you be in a position to help give this a second review and comment on the `goreleaser` changes?

**Comment by phoinixgrr** [comment]
No objections merging this. 🚀 
Tests fails though https://github.com/mattermost/mattermost-govet/actions/runs/9858575330/job/27220277302?pr=39

Additionally, we should consider removing goreleaser in the future. I'll create an internal ticket to address this.

**Comment by enzowritescode** [comment]
@phoinixgrr looks like that is another flag that was also removed. I'll fix it.
https://goreleaser.com/deprecations/#-rm-dist

**Review by phoinixgrr — APPROVED** [review]

**Pull request merged**